test/integration/express.test.js

```diff
@@ -0,0 +1,58 @@
+const Client = require('micro-rpc-client')
+const getPort = require('get-port')
```

Comment:
Great idea to integration tests @esclapes :raised_hands: 

It looks like creating a test service to run RPC requests against is spun up a different way than the rest of the tests.

There's a few examples using the utils with the `test-listen` library: https://github.com/bufferapp/buffer-rpc/blob/master/test/rpc.test.js#L228

It accomplishes the same thing is this code :smile: 

So in effect all of the tests are integration tests -- we're just missing an integration test with the RPC library :wink: